engine/init.asm

```diff
@@ -63,17 +63,11 @@ _Init::
 	call ClearVRAM
 	call ClearSprites
 
-	; Write game version to WRAM.
-	ldh a, [rSVBK]
-	push af
-	ld a, BANK(wGameVersion)
-	ldh [rSVBK], a
-	ld a, HIGH(SAVE_VERSION)
-	ld [wGameVersion], a
-	ld a, LOW(SAVE_VERSION)
-	ld [wGameVersion + 1], a
-	pop af
-	ldh [rSVBK], a
+	; Write checksum to WRAM for save state check
+	ld a, [RomChecksum]
```

Comment:
`RomHeaderChecksum` would be more clear where it's coming from.

Reply:
(Also keeping it distinct from `wRomChecksum` so it doesn't look like a typo; it's unusual to see a load not from Hungarian-notated RAM labels.)